test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo_test_helper
```

Comment:
Looking at the use of this library, it seems to me very weird, as `res.partner` is in base module, so I don't think we need to use `FakeModelLoader`. Can you please check about this for removing unneeded dependencies?